### ISSUE TITLE
longer list for listing the fields shown in the contao lister

### ISSUE THREAD

**Issue by Olli** [contao/contao#6249]
### Description

255 chars obviously are not enough for listing fields of a huge table. So this field need to be way bigger to not have any borders while displaying huge datasets.
Reference: https://github.com/contao/contao/blob/67b4da891e32b2fc460308dddcd7dd0f204a0d41/listing-bundle/contao/dca/tl_module.php#L59

**Comment by aschempp** [comment]
I think 255 chars is plenty for most use cases. If you need more, it's easy to adjust your DCA by changing maxlength and sql in your location application :blush: https://docs.contao.org/dev/reference/dca/fields/

**Comment by Olli** [comment]
Yes I did adjust the DCA.
I don't like this "I think this and that is enough for the most use cases" ... obviously It wasn't enough. This decision is just gut feeling. 
255 byte isn't much and I see no downside in make the field bigger. It doesn't make the system slow and only has advantages. So why not?

**Comment by aschempp** [comment]
Yes it is "gut feeling". Or experience of > 10 years of Contao, whatever you want to call it 😆 

> 255 byte isn't much and I see no downside in make the field bigger.

We have a lot of issues with too many varchar fields, so this **really** is a problem. However, I think we could easily convert this to a `TEXT` field, wdyt @ausi ?

**Comment by ausi** [comment]
> We have a lot of issues with too many varchar fields,…

`tl_module.list_info` is not a varchar field, see: https://github.com/contao/contao/blob/67b4da891e32b2fc460308dddcd7dd0f204a0d41/listing-bundle/contao/dca/tl_module.php#L56-L61

> However, I think we could easily convert this to a `TEXT` field, wdyt @ausi ?

Agree, changing it from `TINYTEXT` to `TEXT` should not be a problem. This would add one byte to the MySQL row size. The InnoDB row size would stay the same.

**Comment by aschempp** [comment]
@Olli would you do a pull request for the discussed changes?
